server/api/app/Applications/User/Controllers/UserController.php

```diff
@@ -156,4 +158,42 @@ public function updateMyProfile(MyProfile $request)
     {
         $this->userService->updateMyProfile($request);
     }
+
+    /**
+     * Handle the avatar upload for the authenticated user.
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function uploadAvatar(Request $request): JsonResponse
+    {
+        try {
+            $userId = Route::current()->parameter('id');
+            $authenticatedUser = Auth::user();
+
+            $userDTO = $this->userService->uploadAvatar($userId, $request, $authenticatedUser);
+
+            return response()->json($userDTO, 200);
+        } catch (ValidationException $e) {
+            return response()->json([
```

Comment:
Use custom validation with a custom FormRequest ex: UploadAvataRequest

Reply:
Can you point to some example ?